README.md

```diff
@@ -69,6 +69,11 @@ from the the project root, issue the following command:
 
     pip install -Ur requirements.txt
 
+###Create test users
+Some of the tests require logging in as an user, and others require logging in as an editor user or paypal user.
+To run these tests you will need to create accounts on https://addons.allizom.org/. If you want to run the tests
+that require editor access, then you will need to ask someone on IRC to upgrade one of your test accounts.
```

Comment:
We might want to add a note about how to create a Paypal user, though I don't really know how we do that. I suspect @krupa can help here.